src/Session.hs

```diff
@@ -173,7 +176,20 @@ isCommand = isPrefixOf commandPrefix
 commandPrefix :: String
 commandPrefix = "-- $> "
 
+isMultilineCommandPrefix :: String -> Bool
+isMultilineCommandPrefix = (==) multilineCommandPrefix
 
+multilineCommandPrefix :: String
+multilineCommandPrefix = "{- $>"
+
+isMultilineCommandSuffix :: String -> Bool
+isMultilineCommandSuffix = (==) multilineCommandSuffix
+
+multilineCommandSuffix :: String
+multilineCommandSuffix = "<$ -}"
+
+wrapGhciMultiline :: [String] -> [String]
+wrapGhciMultiline xs = [":{"] <> xs <> [":}"]
```

Comment:
Can we use `++` for concat as its a bit simpler.

Reply:
sure! bit of a habit from overloaded lists